doc/Changelog.md

```diff
@@ -3,12 +3,16 @@ Changelog {#changelog}
 
 # git master {#master}
 
+* [#53](https://github.com/BlueBrain/Fivox/pull/53)
+  Add support for volume setup via reference volume with 'reference' parameter
+  in volume URI
+  Add 'gidfraction' parameter in volume URI
```

Comment:
Maybe also add (here and/or in the commit message) that the default resolution is now 1 instead of 10